docs/configurations.md

```diff
@@ -0,0 +1,266 @@
+| scalar.db.jdbc.table_metadata.schema                    | Schema name for the table metadata used for ScalarDB.                                                                                                                        | scalardb                     |
+| scalar.db.jdbc.table_metadata.connection_pool.min_idle  | Minimum number of idle connections in the connection pool for the table metadata.                                                                                            | 5                            |
+| scalar.db.jdbc.table_metadata.connection_pool.max_idle  | Maximum number of connections that can remain idle in the connection pool for the table metadata.                                                                            | 10                           |
+| scalar.db.jdbc.table_metadata.connection_pool.max_total | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for the table metadata. Use a negative value for no limit. | 25                           |
+| scalar.db.jdbc.admin.connection_pool.min_idle           | Minimum number of idle connections in the connection pool for admin.                                                                                                         | 5                            |
+| scalar.db.jdbc.admin.connection_pool.max_idle           | Maximum number of connections that can remain idle in the connection pool for admin.                                                                                         | 10                           |
+| scalar.db.jdbc.admin.connection_pool.max_total          | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool for admin. Use a negative value for no limit.              | 25                           |
+
+When you use SQLite3 as a JDBC database, you need to set `scalar.db.contact_points` as follows.
+
+```properties
+scalar.db.contact_points=jdbc:sqlite:your-db.sqlite3?busy_timeout=10000
+```
+
+Unlike other JDBC databases, [SQLite3 does not fully support concurrent access](https://www.sqlite.org/lang_transaction.html).
+In order to avoid frequent errors caused internally by the [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy),
+it is recommended to set a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+
+##### Multi-storage
+
+ScalarDB supports a Multi-storage where you can use multiple storage implementations at the same time.
+You can use Multi-storage by specifying `multi-storage` for the `scalar.db.storage` property.
+
+For the details of the Multi-storage, see [Multi-storage Transactions](multi-storage-transactions.md).
+
+### ScalarDB Server (gRPC)
```

Comment:
Shouldn't we mark it as deprecated? (or recommend users to use ScalarDB Cluster?)
Also, can you add a section for ScalarDB Cluster?

Reply:
Good point. Actually, I was hesitant to mention ScalarDB Cluster here because this is the document of ScalarDB that is a community edition (OSS) product, but ScalarDB Cluster is an enterprise edition product. I think it would be good to discuss the differentiation between the community and enterprise editions. Thank you.

Reply:
As per our offline discussion, I will proceed to mark ScalarDB Server as deprecated and will also add a note about ScalarDB Cluster. Thanks.